### PR TITLE
Pokemon types added.

### PR DESCRIPTION
**Pull request by Artmorse**

### CODE REVIEW

File: src/test/java/com/github/javafaker/PokemonTest.java [hunk 1]
```diff
@@ -16,4 +16,9 @@ public void name() {
     public void location() {
         assertThat(faker.pokemon().location(), matchesRegularExpression("\\w+( \\w+)?( \\w+)?"));
     }
+
+    @Test
+    public void type() {
+        assertThat(faker.pokemon().location(), matchesRegularExpression("\\w+( \\w+)?( \\w+)?"));
```
Comment (AmeniJlassi) [inline src/test/java/com/github/javafaker/PokemonTest.java]: this test is wrong, it should be faker.pokemon().type() and not location()

**Review by AmeniJlassi — CHANGES_REQUESTED** [review]

File: src/test/java/com/github/javafaker/PokemonTest.java (hunk 1 above)
Comment (Artmorse) [reply src/test/java/com/github/javafaker/PokemonTest.java]: you're right, fixed into https://github.com/DiUS/java-faker/pull/775/commits/f319dd1f9e4cbe209d8aa260018d11af4a34766d

Comment (bodiam) [reply src/test/java/com/github/javafaker/PokemonTest.java]: You're pushing into the wrong repo, this one is no longer active. We've forked the project here: https://github.com/datafaker-net/datafaker/, feel free to add contributions, we generally merge everything pretty quick.

**Comment by Artmorse** [comment]
@bodiam It seems the Pokemon Types feature already exists in the https://github.com/datafaker-net/datafaker/ repository.

Can you clarify on the https://github.com/DiUS/java-faker repository that no contributions will be merged? I think you might need to archive this repository.

**Comment by bodiam** [comment]
Nothing has been merged here for a few years, but this isn't my repo, the other is.

**Pull request closed without merging**